lib/prerequisites.sh

```diff
@@ -69,7 +69,8 @@ log INFO "Checking network access ($BROKER_URL_FOR_PREREQ) ..."
 source "${CONFIG_DIR}${PROJECT}".conf
 source ${PROJECT}/vars
 
-if [ "${PROJECT}" != "minimal" ]; then
+#if [ "${PROJECT}" != "minimal" ]; then
+if false; then
```

Comment:
Please revert that. This was only present to allow the bridgehead to run without any beam connection at all during development.